src/server.js

```diff
@@ -0,0 +1,80 @@
+// Express file
+const express = require("express");
+const exphbs = require("express-handlebars");
+const path = require("path");
+const morgan = require("morgan");
+const methodOverride = require("method-override");
+const session = require("express-session");
+const flash = require("connect-flash");
+
+// Initializations
+const app = express();
+
+// Settings
+app.set("port", process.env.PORT || 4000);
+/*app.set("views", path.join(__dirname, "views"));
+app.engine(
```

Comment:
is all of this commented code something that will be used? otherwise don't see the point on leaving it here.